src/Cli/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -70,27 +79,45 @@ public static TestCommand FromParseResult(ParseResult result, string[] settings,
             if (!hasRootVariable)
             {
                 testCommand.EnvironmentVariable(rootVariableName, rootValue);
+                VSTestTrace.WriteTrace($"Root variable set {rootVariableName}:{rootValue}");
             }
 
+            VSTestTrace.SafeWriteTrace(() => $"Starting test using MSBuild with arguments '{testCommand.GetArgumentsToMSBuild()}' custom MSBuild path '{msbuildPath}' norestore '{noRestore}'");
```

Comment:
Could it be worth to split the line? Something like:

```
Starting test using MSBuild with
- arguments: `testCommand.GetArgumentsToMSBuild()`
- MSBuild path: `msbuildPath`
- norestore: `noRestore`
```

Reply:
It's log for "emergency" we don't need beautiful formatting. By design is disabled and I hope that we never need to use it 😄